addons/test_website_slides_full/tests/test_ui_wslides.py

```diff
@@ -4,66 +4,21 @@
 from dateutil.relativedelta import relativedelta
 from odoo.fields import Datetime
 from odoo import tests
-from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests.common import users
 from odoo.addons.website_slides.tests.test_ui_wslides import TestUICommon
 
 
 @tests.common.tagged('post_install', '-at_install')
-class TestUi(AccountTestInvoicingCommon, TestUICommon):
+class TestUi(TestUICommon):
 
-    def test_course_certification_employee(self):
-        user_demo = self.user_demo
-        self.user_demo.write({
-            'company_id': self.env.company.id,
-            'company_ids': [(6, 0, self.env.company.ids)],
-        })
-        self.user_demo.sudo().partner_id.company_id = self.env.company
-        # Avoid Billing/Shipping address page
-        user_demo.write({
-            'groups_id': [(5, 0), (4, self.env.ref('base.group_user').id)],
-            'street': '215 Vine St',
-            'city': 'Scranton',
-            'zip': '18503',
-            'country_id': self.env.ref('base.us').id,
-            'state_id': self.env.ref('base.state_us_39').id,
-            'phone': '+1 555-555-5555',
-            'email': 'admin@yourcompany.example.com',
-        })
-
-        # Specify Accounting Data
-        cash_journal = self.env['account.journal'].create({'name': 'Cash - Test', 'type': 'cash', 'code': 'CASH - Test'})
-        self.env.ref('website.default_website').company_id = self.env.company
-        self.env['payment.provider'].sudo().search([('code', '=', 'demo')]).write({
-            'journal_id': cash_journal.id,
-            'state': 'test',
-            'website_id': self.env.ref('website.default_website').id,
-            'company_id': self.env.company.id,
-        })
-        a_recv = self.env['account.account'].create({
-            'code': 'X1012',
-            'name': 'Debtors - (test)',
-            'reconcile': True,
-            'account_type': 'asset_receivable',
-        })
-        a_pay = self.env['account.account'].create({
-            'code': 'X1111',
-            'name': 'Creditors - (test)',
-            'account_type': 'liability_payable',
-            'reconcile': True,
-        })
+    def setUp(cls):
```

Comment:
We usually always use "setUpClass", even if in this case it should not change anything, mostly best practice.

Reply:
I understand but, we're using this `TestUICommon` and they used `setUp` , See [Here](https://github.dev/odoo/odoo/blob/a56f395e9c012d9b4afa342ab2c4826b23083461/addons/website_slides/tests/test_ui_wslides.py#L15-L18)

Reply:
Fair enough, you can keep it then.